client/my-sites/migrate/section-migrate.jsx

```diff
@@ -225,22 +225,32 @@ class SectionMigrate extends Component {
 	}
 
 	renderMigrationProgress() {
-		const { sourceSite, targetSite } = this.props;
+		const { sourceSite, targetSite, translate } = this.props;
 		const sourceSiteDomain = get( sourceSite, 'domain' );
 		const targetSiteDomain = get( targetSite, 'domain' );
+		const subHeaderText = translate(
```

Comment:
Thanks for considering the translatability of the strings, but I really thing we should wait to actually wrap them in `translate` calls until the strings are finalized. See p2JRYi-4gH-p2